person.rb

```diff
@@ -0,0 +1,41 @@
+require './interface'
+require './capitalize_decor'
+require './trim_decor'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_writer :id
+  attr_reader :rentals
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def correct_name
+    @name
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] In the Person class a method for adding rentals should be present, kindly make sure to create a method `add_rentals` that makes use of the `Rental` class to create a new rental